### ISSUE TITLE
pip3 install pyhmy returns outdated version

### ISSUE THREAD

**Issue by mirrormirage0** [harmony-one/pyhmy#21]
Manually cloning the repo and doing a make install works fine.

However, doing a pip3 install pyhmy gives a version without some essential features. e.g. from pyhmy import signing fails

**Comment by DeFiAccount** [comment]
yo can you plz share the code for what you did?

i cloned the repo with:
```
git clone https://github.com/harmony-one/pyhmy.git
cd pyhmy/
make install
```
but i still can't get a few things to import correctly, like these:

```
from pyhmy.validator import Validator
from pyhmy import signing
from pyhmy import staking_structures, staking_signinge
```
none of these 3 are importing. any help is appreciated, thanks :)

**Comment by ChristianWilkie** [comment]
@DeFiAccount - maybe you're using a venv? instead of make install, try "pip install /path/to/local/pyhmy" 

also - related thread on the harmony community forum https://talk.harmony.one/t/ci-cd-for-pyhmy-to-pypi-repository/5923

**Comment by johnashu** [comment]
> @DeFiAccount - maybe you're using a venv? instead of make install, try "pip install /path/to/local/pyhmy"
> 
> also - related thread on the harmony community forum https://talk.harmony.one/t/ci-cd-for-pyhmy-to-pypi-repository/5923

Yes, this is how I installed but also you will need tom create an `__init__.py` file in the bech32 folder as detailed here..

https://github.com/harmony-one/pyhmy/pull/24/files

**Comment by MaxMustermann2** [comment]
Reposting from https://github.com/harmony-one/pyhmy/issues/25#issuecomment-1226162905

`pip install pyhmy` now returns an updated version 0.1.0 by default. If you have any existing versions, please remove them (`pip uninstall pyhmy`) and make sure to clear your cache (`pip cache purge`).

You can see available versions
```bash
$ pip index versions pyhmy
pyhmy (0.1.0)
Available versions: 0.1.0, 0.0.0
  INSTALLED: 0.1.0
  LATEST:    0.1.0
```
